Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Memberships.cs

```diff
@@ -15,6 +15,11 @@ public Memberships()
         public string MembershipType { get; set; }
```

Comment:
Usually its better to make "type" variables Enums, it makes it more robust because you only allow it to be a certain set of values

Reply:
I was advised on this, have Enums on a to read list and will update accordingly